docker/roop-docker.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```

Comment:
will also not land in our repo... I don't understand most of it and whatever it does can be done via a shared volume to the host I guess